src/SWP/Bundle/ContentBundle/Doctrine/ODM/PHPCR/Article.php

```diff
@@ -77,4 +93,20 @@ public function getChildren()
     {
         return $this->children;
     }
+
+    /**
+     * @return Collection
+     */
+    public function getMedia()
```

Comment:
this method is not forced by any interface, should be added to ArticleInterface or maybe add MediaInterface and extend ArticleInterface by MediaInterface